apps/parsley/src/components/SectionsFeatureModal/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import { useMatch } from "react-router-dom";
+import { useSectionsFeatureDiscoveryAnalytics } from "analytics/sectionsFeatureDiscovery/useSectionsFeatureDiscoveryAnalytics";
+import routes from "constants/routes";
+import { zIndex } from "constants/tokens";
+import { useSectionsFeatureDiscoveryContext } from "context/SectionsFeatureDiscoveryContext";
+import { useParsleySettings } from "hooks/useParsleySettings";
+import { releaseSectioning } from "utils/featureFlag";
+import { graphic } from "./graphic";
+
+export const SectionsFeatureModal = () => {
+  const { closeFeatureModal, isOpenFeatureModal } =
+    useSectionsFeatureDiscoveryContext();
+  const isViewingTaskLog = useMatch(routes.evergreenLogs);
```

Comment:
Any reason not to use the value from `useLogContext`

Reply:
It's possible to import `useLogContext` and get the same result because the route is 1:1 with the logType. I opted to use the route matching because it's makes it simpler to understand how/when the modal is rendered compared to doing a logType comparison. I switched over to `useLogContext` because that has a benefit of decoupling routing from the rest of the components.